packages/connect-core/src/connections/IOrganizationConnector.ts

```diff
@@ -9,6 +9,7 @@ import Role from '../entities/Role'
 export default interface IOrganizationConnector {
   readonly name: string
   readonly network: Network
+  readonly config: any
```

Comment:
I couldn’t use `object` or `{}` because TS will enforce that they stay empty… but I found another way :smile: e78a90ed826659c389477368461fb893dc0230d1